src/main/kotlin/step3/racingcar/domain/CurrentTurn.kt

```diff
@@ -0,0 +1,20 @@
+package step3.racingcar.domain
+
+class CurrentTurn(var _currentTurn: Int = 0) {
+    val value: Int
+        get() = _currentTurn
```

Comment:
```suggestion
class CurrentTurn(currentTurn: Int = 0) {
    var value: Int = currentTurn
        private set
```
내부에서만 변경할 수 있도록 이렇게 작성할 수도 있어요

Reply:
아 이런 방식으로도 구현이 가능한가 보네요
위와 같이 선언하면 `currentTurn` 파라미터 값은 별도의 프로퍼티(백킹 필드)가 생기지 않고 1회성으로만 사용되는것이 맞나요?

Reply:
네 맞습니다

Reply:
전보다 더 깔끔한 느낌이네요! 수정하겠습니다